Tilers/CityTiler/citym_cityobject.py

```diff
@@ -214,25 +236,65 @@ def retrieve_geometries(cursor, city_object_ids, offset, objects_type):
         """
         city_object_ids_arg = str(city_object_ids).replace(',)', ')')
 
-        cursor.execute(objects_type.sql_query_geometries(offset,
+        cursor.execute(objects_type.sql_query_geometries_with_texture_coordinates(offset,
```

Comment:
Would it be hard to keep the original version of retrieve_geometries() and to
add a retrieve_coordinate_textures() that would call retrieve_geometries() or
be somehow articulated with retrieve_geometries()  ?
Separation of concerns (refer e.g. to https://en.wikipedia.org/wiki/Separation_of_concerns )
is a powerful design force...